### PR TITLE
Feature/course detail screen

### PR DESCRIPTION
**Pull request by Humnaa**
Trello Link : https://trello.com/c/uw7XZ3fG/87-course-detail-screen

### CODE REVIEW

File: src/containers/Dashboard/Courses/Card.js [hunk 1]
```diff
@@ -1,28 +1,50 @@
...
             <div className="header">
               <h5 className="card-title">{this.props.title}</h5>
             </div>
-            <div className="card-content">{courseList}</div>
+            <div className="card-content">
+              {this.props.data.map((course, index) => (
+                <div
+                  className="course-name"
+                  key={index}
+                  onClick={() => this.getCourseId(course.courseId, course.name)}
```
Comment (ayousaf11) [inline src/containers/Dashboard/Courses/Card.js]: @Humnaa why is this method name is `getCourseID` ?

File: src/containers/Pages/CoursePage.js [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+      );
+    } else {
+      tutorials =
+        this.props.tutorials &&
+        this.props.tutorials.map((tutorial, index) => (
+          <div
+            className="tutorial-name"
+            key={index}
+            onClick={() =>
+              this.getTutorialId(tutorial.tutorialId, tutorial.name)
```
Comment (ayousaf11) [inline src/containers/Pages/CoursePage.js]: @Humnaa 
why is this method named `getTutorialId`?

File: src/containers/Pages/CoursePage.js [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+        </div>
+      );
+    } else {
+      quizes =
+        this.props.quizes &&
+        this.props.quizes.map((quiz, index) => (
+          <div
+            className="tutorial-name"
+            key={index}
+            onClick={() => this.getQuizId(quiz.quizId)}
```
Comment (ayousaf11) [inline src/containers/Pages/CoursePage.js]: @Humnaa 
why is this method named `getQuizId`?

**Review by ayousaf11 — CHANGES_REQUESTED** [review]
If there is no update in package.json then you should also not commit the package-lock.json
Remove the jpeg image if not used.
@Humnaa

File: src/containers/Pages/CoursePage.js (hunk 2 above)
Comment (HarisShahidHameed) [reply src/containers/Pages/CoursePage.js]: I have renamed it `getTutorialById`

File: src/containers/Dashboard/Courses/Card.js (hunk 1 above)
Comment (HarisShahidHameed) [reply src/containers/Dashboard/Courses/Card.js]: I have renamed it getCourseById

File: src/containers/Pages/CoursePage.js (hunk 3 above)
Comment (HarisShahidHameed) [reply src/containers/Pages/CoursePage.js]: I have renamed it getQuizById

**Review by ayousaf11 — APPROVED** [review]

**Pull request merged**